android/src/main/java/com/swmansion/rnscreens/ScreenStack.java

```diff
@@ -169,65 +169,63 @@ protected void performUpdate() {
       }
     }
 
-    boolean customAnimation = false;
+    // variables used for "open/back animation" logic
+    boolean shouldPlayOpenAnimation = true;
+    boolean isCustomAnimation = false;
+    int transition;
```

Comment:
Would adding it a default value (`FragmentTransaction.TRANSIT_FRAGMENT_OPEN` if following the convention) destroy any of the logic? If not, I think we should do it.